src/Compiler/CompilerPasses.hpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------------- CompilerPasses.hpp -------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Functions for adding passes.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#include "mlir/Pass/PassManager.h"
+
+void addONNXToMLIRPasses(mlir::PassManager &pm);
```

Comment:
Can we place these declaration in the onnx_mlir namespace as well ?

Reply:
Done.